web/tests/lib/test_stream.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+exports.make_stream = (opts = {}) => {
+    const default_stream = {
+        audible_notifications: false,
+        can_remove_subscribers_group: 0,
+        color: "blue",
+        creator_id: null,
+        date_created: Date.now(),
+        description: "A Scandinavian country",
+        desktop_notifications: false,
+        email_address: "test@example.com",
+        email_notifications: false,
+        first_message_id: null,
+        history_public_to_subscribers: true,
+        invite_only: false,
+        is_announcement_only: false,
+        is_muted: false,
+        is_web_public: true,
+        message_retention_days: null,
+        name: "Denmark",
```

Comment:
Also helpful is if the default name is different each time when the function is repeatedly called. For example make the name random:
https://github.com/zulip/zulip-mobile/blob/2ae8d8e54dcf25ce760206b0f6fc16923b4375b5/src/__tests__/lib/exampleData.js#L310-L314

or make related properties like name, email, description, etc. use a common random string by default:
https://github.com/zulip/zulip-mobile/blob/2ae8d8e54dcf25ce760206b0f6fc16923b4375b5/src/__tests__/lib/exampleData.js#L163-L164